iset.mm

```diff
@@ -15856,11 +15856,11 @@ Predicate calculus with distinct variables (cont.)
     sb7f.1 $e |- ( ph -> A. z ph ) $.
     $( This version of ~ dfsb7 does not require that ` ph ` and ` z ` be
        distinct.  This permits it to be used as a definition for substitution
-       in a formalization that omits the logically redundant axiom ~ ax-17 i.e.
-       that doesn't have the concept of a variable not occurring in a wff.
-       ( ~ df-sb is also suitable, but its mixing of free and bound variables
-       is distasteful to some logicians.)  (Contributed by NM, 26-Jul-2006.)
-       (Proof shortened by Andrew Salmon, 25-May-2011.) $)
+       in a formalization that omits the logically redundant axiom ~ ax-17 ,
+       i.e., that does not have the concept of a variable not occurring in a
+       formula.  (Definition ~ df-sb is also suitable, but its mixing of free
+       and bound variables is distasteful to some logicians.)  (Contributed by
+       NM, 26-Jul-2006.)  (Proof shortened by Andrew Salmon, 25-May-2011.) $)
```

Comment:
Should now be ~ dfsb1 ("Definition" not necessary? I personally read a label `dfnoun` as "Definition noun" in my head)

Same for nf.mm and set.mm, sb7f and sb7h

Reply:
It is already correct in set.mm, and dfsb1 does not exist in iset.mm and nf.mm.  Therefore, I deleted that parenthetical remark (since obviously df-sb is "suitable" since this is the actual definition).

Reply:
I mean, it might not be obvious when one is not familiar with the variants of df-sb, in which case the link might be useful. It's incosistent that set.mm has the remark too, but I'll approve anyway since this isn't very significant compared to the ~ df-sb vs ~ dfsb1 typo.

Reply:
What is inconsistent is actually the differing definitions df-sb among these databases, but that's for a future MR (on which I have currently no plans).